claim_ai/evaluation/ai_model.py

```diff
@@ -0,0 +1,129 @@
+import os
+from pathlib import Path
+
+import numpy as np
+from datetime import datetime
+
+import pandas
+import pickle
+import joblib
+
+from claim_ai.apps import ClaimAiConfig
+
+
+class AiModel:
+
+    def __init__(self):
+        self.model = self._load_model()
+        self.encoder = self._load_encoder()
+        self.scaler = self._load_scaler()
+
+    def evaluate_bundle(self, input_bundle):
+        index, clean_input = self.sanity_check(input_bundle)
+        clean_input = self.fill_missing_varaibles(clean_input)
+        clean_input = self.convert_variables(clean_input)
+        clean_input = self.normalize_input(clean_input)
+        return index, self.predict(clean_input)
+
+    def sanity_check(self, input):
+        date_from = datetime(2016, 5, 15)
+        date_claimed = datetime(2016, 5, 15)
```

Comment:
please add date in the configuration with default value 01.01.2016. only one date for both dates here.

Reply:
Date changed to be retrieved from the configuration